dts/bindings/display/solomon,ssd1306fb-common.yaml

```diff
@@ -49,6 +49,10 @@ properties:
       If connected directly the MCU pin should be configured
       as active low.
 
+  sh1106-compatible:
+    type: boolean
+    description: Indicate device is compatible with SH1106
```

Comment:
Why not adding a proper and usual compatible property to do that?
That's the point of compatible properties and if required several compatibles can be provided.

Reply:
I fixed it detecting with dts compatibility.